### ISSUE TITLE
github is no longer in sync with what is deployed from the solutions page

### ISSUE THREAD

**Issue by mjtk-ktjm** [amazon-connect/ai-powered-speech-analytics-for-amazon-connect#9]
I notice that the agentAssist.html deployed via the cloudformation stack does not match what is in github. Is there an eta for getting these in sync?

**Comment by efh365** [comment]
We just updated both the GitHub repo and the Solutions page yesterday. When did you last deploy the stack?

**Comment by mjtk-ktjm** [comment]
hmm.. I deployed a fresh stack yesterday following up on an issue that was reported with getOverallSentiment behavior. But when I cloned the repo this morning, I was looking at a different version of the file. Maybe more of a github issue than a code issue?

Though, I do have a follow up question on that.
If this is the current code for getOverallSentiment :

```
   3         function getOverallSentiment(text, language, endOfCall){
   2   ┊ ┊ ┊ ┊ ┊ ┊ ┊ ┊ ┊ var space = " ";
   1   ┊ ┊ ┊ ┊ ┊ ┊ ┊ ┊ ┊ entireTranscript = entireTranscript += space += text;
1062   ┊ ┊ ┊ ┊ ┊ ┊ ┊ ┊ ┊ language = language ? language : "en-US";
   1   ┊ ┊ ┊ ┊ ┊ ┊ ┊ ┊ ┊ $("#section-agent").show("slide");
   2
   3   ┊ ┊ ┊ ┊ ┊ ┊ ┊ ┊ ┊ if (text) {
   4             if(entireTranscript.length > 0){
   5               var source = entireTranscript;
```
then `language = language ? language : "en-US";` would seem to be a regression. 

en-US is not a valid language code for the service ('en' is). The other version I was looking at was using the fromLanguageDict variable to replace en-US with en in the same function.

Further, the `entireTranscript = entireTranscript += space += text;` was in processText instead of getOverallSentiment.

Maybe my wires are just getting crossed somehow.
